### DISCUSSION TITLE
Command failed with exit code 16 (EBUSY) when attempting to deploy a Ruby application

### DISCUSSION THREAD

**Discussion by Jomy10** [vercel/community#2333] [Bug Report]
When trying to deploy a Vercel servererless function written in Ruby, I get the following output:
```
❯ vercel deploy
Vercel CLI 29.1.1
🔍  Inspect: https://vercel.com/ksa-de-biekorf/project-name/ID [3s]
Error: Command failed with exit code 16 (EBUSY): /ruby27/bin/bundler install --deployment --gemfile /vercel/path0/Gemfile --path /vercel/path0/vendor/bundle
```

## Steps to reproduce

1. Create the following folder structure
```
.                                                                                                                 
├── Gemfile
└── api                                                                                                           
    └── hello.rb
```
In hello.rb a simple function:
```ruby
Handler = Proc.new do |req, resp|
  name = req.query['name'] || "World"

  resp.status = 200
  resp['Content-Type'] = 'text/text; charset=utf8'
  resp.body = "Hello #{name}!"
end
```

In the Gemfile:
```ruby
source "https://rubygems.org"

ruby "~> 2.7.x"
```
2. run `vercel deploy` and follow the steps to set up a project
3. The error will now appear

Any help on this issue?

**Comment by 1gn0r4nd** [comment]
I seem to get the same issue, and I copied the index.rb from their page. Added the lockfile to the repo too, can't figure out what else I could be missing.

**Comment by pdsullivan** [comment]
i am having the exact same issue with the code provided on the example. this command runs locally fine `bundler install --deployment` but when deploying on vercel it gets the error of `Error: Command failed with exit code 16 (EBUSY): /ruby27/bin/bundler install --deployment --gemfile /vercel/path0/Gemfile --path 
 /vercel/path0/vendor/bundle`
